packages/xrpl/test/client/autofill.test.ts

```diff
@@ -28,25 +36,31 @@ describe('client.autofill', function () {
       Sequence,
       LastLedgerSequence,
     }
-    const txResult = await this.client.autofill(tx)
+    const txResult = await testContext.client.autofill(tx)
 
     assert.strictEqual(txResult.Fee, Fee)
     assert.strictEqual(txResult.Sequence, Sequence)
     assert.strictEqual(txResult.LastLedgerSequence, LastLedgerSequence)
   })
 
-  it('converts Account & Destination X-address to their classic address', async function () {
+  it('converts Account & Destination X-address to their classic address', async () => {
     const tx: Payment = {
       TransactionType: 'Payment',
       Account: 'XVLhHMPHU98es4dbozjVtdWzVrDjtV18pX8yuPT7y4xaEHi',
       Amount: '1234',
       Destination: 'X7AcgcsBL6XDcUb289X4mJ8djcdyKaB5hJDWMArnXr61cqZ',
     }
-    this.mockRippled.addResponse('account_info', rippled.account_info.normal)
-    this.mockRippled.addResponse('server_info', rippled.server_info.normal)
-    this.mockRippled.addResponse('ledger', rippled.ledger.normal)
+    testContext.mockRippled?.addResponse(
```

Comment:
This could quietly not add the response if `mockRippled` ends up not being defined. Maybe it'd be better to define two `XrplTestContext`'s - one where you have `mockRippled` for sure, and one where you are connected to a standalone node? 

That way if your mock environment doesn't get setup correctly - it can just crash and we can avoid these requests passing quietly. (It could be that none of the tests succeed if `rippled` does nothing, but that seems like something easy to overlook in one test case out of the hundreds)

Reply:
I changed to use `testContext.mockRippled!.addResponse`